### PR TITLE
Made some changes to consolidate the jquery tools

### PR DESCRIPTION
**Pull request by teryhill**
Made some changes to consolidate the jquery tools #416 

When complete thru the code base the added items will be removed from /library/js.

### CODE REVIEW

File: interface/billing/edit_payment.php [hunk 1]
```diff
@@ -477,30 +478,17 @@
...
-<script type="text/javascript" src="../../library/dialog.js"></script>
-<script type="text/javascript" src="../../library/js/jquery.1.3.2.js"></script>
-<script type="text/javascript" src="../../library/js/fancybox/jquery.fancybox-1.2.6.js"></script>
-<script language='JavaScript'>
- var mypcc = '1';
-</script>
 <?php include_once("{$GLOBALS['srcdir']}/payment_jav.inc.php"); ?>
 <?php include_once("{$GLOBALS['srcdir']}/ajax/payment_ajax_jav.inc.php"); ?>
-<script type="text/javascript" src="../../library/js/common.js"></script>
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/common.js"></script>
```
Comment (aethelwulffe) [inline interface/billing/edit_payment.php]: Pretty sure common.js and some others could be moved already,  but if you are sure that this list of includes are everything that is required and intend on migrating other bits in other commits (good idea) then this is fine.

File: interface/globals.php [hunk 2]
```diff
@@ -191,13 +191,13 @@ function undoMagicQuotes($array, $topLevel=true) {
 //module config TODO:  module and global registry for same.
 
 // images directory
-$GLOBALS['images_path'] = "$web_root/assets/images";
+$GLOBALS['images_path'] = "$web_root/assets/images/";
 
 // css directory
-$GLOBALS['css_path'] = "$webserver_root/assets/css";
```
Comment (aethelwulffe) [inline interface/globals.php]: do we have $webserver_root and $web_root both still?

File: interface/orders/procedure_provider_edit.php [hunk 3]
```diff
@@ -44,7 +44,7 @@ function invalue($name) {
 <script type="text/javascript" src="<?php echo $webroot ?>/interface/main/tabs/js/include_opener.js"></script>    
 <title><?php echo $ppid ? xlt('Edit') : xlt('Add New') ?> <?php echo xlt('Procedure Provider'); ?></title>
 <link rel="stylesheet" href='<?php echo $css_header ?>' type='text/css'>
-<script type="text/javascript" src="../../library/js/jquery.1.3.2.js"></script>
+<script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>/jquery-min-3-1-1/index.js"></script>
```
Comment (aethelwulffe) [inline interface/orders/procedure_provider_edit.php]: Assuming 3.1.1 is tested...

File: interface/reports/encounters_report.php [hunk 4]
```diff
@@ -53,8 +54,8 @@ function show_doc_total($lastdocname, $doc_encounters) {
   }
 }
 
-$form_from_date = fixDate($_POST['form_from_date'], date($DateFormat));
-$form_to_date = fixDate($_POST['form_to_date'], date($DateFormat));
+$form_from_date = fixDate($_POST['form_from_date'], date('Y-m-d'));
```
Comment (aethelwulffe) [inline interface/reports/encounters_report.php]: is there a problem with the configurable date format here that we need a static Y-m-d?

File: interface/billing/edit_payment.php (hunk 1 above)
Comment (teryhill) [inline interface/billing/edit_payment.php]: Tried to move the ones to header.inc.php that were common to all files. Not sure what opening the unused ones would benefit or hinder.

File: interface/globals.php (hunk 2 above)
Comment (teryhill) [inline interface/globals.php]: Nope we do not use $webserver_root that I can find

File: interface/orders/procedure_provider_edit.php (hunk 3 above)
Comment (teryhill) [inline interface/orders/procedure_provider_edit.php]: Yep have it running and testing locally.

File: interface/reports/encounters_report.php (hunk 4 above)
Comment (teryhill) [inline interface/reports/encounters_report.php]: I was getting a goofy looking date when I was in MM/DD/YYYY format so I switched it back till I could spend some time later looking into it.

**Comment by aethelwulffe** [comment]
It is all better now then, as soon as someone merges this commit @tmccormi 
I am with ken on most of this stuff.
Being able to just make viewable objects available to anything in the application would be the way to go with (at very least) the super stuff.
  It would be almost as good as having a real programming language.
...anyway, I would still be interested in figuring out why superglobals act as superglobals on my machine, and not with Terry's set-up.  Probably that I have something turned on in my server variables that is a no-no for production.

**Pull request merged**